src/backend/base/langflow/components/vectorstores/astradb.py

```diff
@@ -251,6 +176,12 @@ class AstraDBVectorStoreComponent(LCVectorStoreComponent):
             info="Boolean flag to determine whether to ignore invalid documents at runtime.",
             advanced=True,
         ),
+        NestedDictInput(
+            name="astradb_vectorstore_kwargs",
+            display_name="AstraDBVectorStore Parameters",
```

Comment:
You can add a default dict here with the params of your choice if you like

Reply:
Great point! All the defaults are None, but maybe it would help users see the options by explicitly showing that